src/MimeMapping/MimeMapping.csproj

```diff
@@ -41,22 +39,12 @@ https://learn.microsoft.com/dotnet/api/system.web.mimemapping.getmimemapping
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Remove="Test\**" />
-    <EmbeddedResource Remove="Test\**" />
-    <None Remove="Test\**" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Remove=".editorconfig" />
-    <None Remove=".gitignore" />
-    <None Remove="LICENSE.md" />
-    <None Remove="**/TestResults/**" />
-    <None Include="README.md" Pack="true" PackagePath="\" />
+    <None Include="../../README.md" Pack="true" PackagePath="\" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
-    <PackageReference Include="MinVer" Version="4.3.0" PrivateAssets="All" />
+    <PackageReference Include="MinVer" Version="5.0.0" PrivateAssets="All" />
```

Comment:
wrong merge , don't change dependency versions in this pr

Reply:
Manual merge to fix upstream conflicts. It's actual version. See https://github.com/zone117x/MimeMapping/blob/master/MimeMapping.csproj